src/test/java/io/vertx/ext/shell/command/base/DeployVerticleTest.java

```diff
@@ -0,0 +1,143 @@
+package io.vertx.ext.shell.command.base;
+
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.Vertx;
+import io.vertx.ext.shell.Shell;
+import io.vertx.ext.shell.ShellServer;
+import io.vertx.ext.shell.system.ExecStatus;
+import io.vertx.ext.shell.system.Job;
+import io.vertx.ext.shell.term.Pty;
+import io.vertx.ext.unit.Async;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author <a href="mailto:emad.albloushi@gmail.com">Emad Alblueshi</a>
+ */
+
+@RunWith(VertxUnitRunner.class)
+public class DeployVerticleTest {
```

Comment:
create a single private testDeploy method that is called by the other methods, so we don't repeat several times the same similar code.